kmsauth/__init__.py

```diff
@@ -57,6 +57,9 @@ def __init__(
             endpoint_url=None,
             token_cache_size=4096,
             stats=None,
+            max_pool_connections=100,
+            connect_timeout=1,
+            read_timeout=1,
             ):
         """Create a KMSTokenValidator object.
 
```

Comment:
we should probably also document these new parameters

Reply:
added